Resources/Private/Templates/Carousel/Show.html

```diff
@@ -24,6 +24,7 @@
                         <h3>
                             <f:format.nl2br>{slide.title}</f:format.nl2br>
                         </h3>
+                        <p><f:format.html>{slide.sliderText}</f:format.html></p>
```

Comment:
- Add if condition
- ```f:format.html()``` generates own ```<p class="bodytext">``` wrapper

This will result into:
```html
<f:if condition="{slide.description}">
    {slide.description -> f:format.html()}
</f:if>
```